sdk/typescript/package.json

```diff
@@ -49,6 +50,11 @@
 			"source": "./src/faucet/index.ts",
 			"import": "./dist/esm/faucet/index.js",
 			"require": "./dist/cjs/faucet/index.js"
+		},
+		"./client": {
```

Comment:
Is it api or client? The `files` say `api` but this says client but both exist?

Reply:
Good catch, it was originally API, but I think client makes more sense as the name of the export